api/control/v1alpha1/types.go

```diff
@@ -194,74 +194,54 @@ type Binding struct {
 // of a BindingPolicy's `what` and `where`: what objects to propagate and to where.
```

Comment:
This is a bit outdated, does not cover the inclusion of ResourceVersion.

Reply:
Should the resource version really be mentioned here? A resource version is a part of an object, it is part of what identifies a specific `what`, that is an object.

Reply:
But it's fine, and I'll update it.

Reply:
But what is here is not _objects_, instead it is _object references_ --- plus ResourceVersions.